packages/perseus-editor/src/components/util.ts

```diff
@@ -215,3 +222,34 @@ export const getSinusoidEquation = (startCoords: [Coord, Coord]) => {
         verticalOffset.toFixed(3)
     );
 };
+
+export const getQuadraticEquation = (startCoords: [Coord, Coord, Coord]) => {
+    const p1 = startCoords[0];
+    const p2 = startCoords[1];
+    const p3 = startCoords[2];
+
+    const denom = (p1[0] - p2[0]) * (p1[0] - p3[0]) * (p2[0] - p3[0]);
+    if (denom === 0) {
+        // Many of the callers assume that the return value is always defined.
+        return "Division by zero error";
+    }
+    const a =
+        (p3[0] * (p2[1] - p1[1]) +
+            p2[0] * (p1[1] - p3[1]) +
+            p1[0] * (p3[1] - p2[1])) /
+        denom;
+    const b =
+        (p3[0] * p3[0] * (p1[1] - p2[1]) +
+            p2[0] * p2[0] * (p3[1] - p1[1]) +
+            p1[0] * p1[0] * (p2[1] - p3[1])) /
+        denom;
+    const c =
+        (p2[0] * p3[0] * (p2[0] - p3[0]) * p1[1] +
+            p3[0] * p1[0] * (p3[0] - p1[0]) * p2[1] +
+            p1[0] * p2[0] * (p1[0] - p2[0]) * p3[1]) /
+        denom;
```

Comment:
This is copied from `InteractiveGraph.getQuadraticCoefficients`?

I wish we had a central place where this logic could live, and both `perseus` and `perseus-editor` could import it from there. I think `kmath` is probably the best candidate that place. Would it be too hard to move `getQuadraticCoefficients` to `kmath`?

If you don't want to move it in this PR, I can move it once this is landed.

Reply:
Yes, that would be awesome! Sorry, I didn't see your comment before landing.